client/src/contexts/AuthContext.js

```diff
@@ -15,6 +15,8 @@ const reducer = (state, action) => {
       return { user: action.payload.user, errorMessage: "" };
     case "clearErrorMessage":
       return { ...state, errorMessage: "" };
+    case "refreshUser":
```

Comment:
You can remove this. Merge with the below branch and call refreshLoggedInUser on the auth context
https://github.com/hatchways/team-bluejay/tree/kareem-jwt-cookie-authentication-%2315-%2316